CMS-Enterprise/easi-app
# EASI-4551 Add a system team member

# EASI-4551
https://www.figma.com/design/z8GeZPDfIh9oKfHK1cX7Fo/System-centric-EASi?node-id=1008-9405&node-type=frame&t=HoeR8NMkJqIvmowW-0



## Description

- The team member form now has a workspace mode
  - `useIsWorkspaceParam` will check for the `?workspace` mode
- The system profile team form remains the same

## How to test this change

- Make sure the `systemWorkspaceTeam` flag is on
- Go to the system workspace team card
- Add a member to see the changes

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
